src/auth/auth.service.ts

```
@@ -88,11 +88,12 @@ export class AuthService {
 
   async join(createUserDto: CreateUserDto) {
-    const newUser = await this.userService.create(createUserDto);
-    return {
-      accessToken: this.generateAccessToken(newUser),
-      refreshToken: await this.generateRefreshToken(newUser.id),
-    };
+    try {
+      await this.userService.create(createUserDto);
+      return 'sign ok';
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)